README.md

```diff
@@ -173,8 +173,8 @@ Returns a function which accepts an input value to be validated. This function t
 **Parameters**
 
 - `rootValidator`: The root validator to assert values with.
-- `options`: An options object.
-- `options.apiName`: String to prefix every error message with.
+- `options`: An options object or a string. If it is a string, it will be interpreted as `options.description`.
+- `options.description`: A string to prefix every error message with. For example, if `description` is `myFunc` and a string is invalid, the error message with be `myFunc: value must be a string`. (Formerly `options.apiName`, which works the same but is deprecated.)
```

Comment:
It would be great if you could also update the example below to reflect the updated usage.